### ISSUE TITLE
Feature Request: Win Probability Based Variations

### ISSUE THREAD

**Issue by cooperstevenson** [pnprog/goreviewpartner#35]
Hi,

Thanks for 'Go Review Partner', I use it all the time! Is a feature to perform variations based on a significant rise/drop in win probability rate during the game an easy thing to do?

**Comment by pnprog** [comment]
Hi!
Do you mean keeping only the variations produces by the bot when the actual game win rate drops higher than a certain value?

**Comment by cooperstevenson** [comment]
Yes, exactly. For example, if black's win ratio goes up or down by (a presumably user-configurable) 5 per cent then make a variation.
